Sources/GRPCInProcessTransport/Client.swift

```diff
@@ -0,0 +1,359 @@
+/*
```

Comment:
Can you rename this file? Normally when you extend a type you name the file `<ThingBeingExtended>+<WhatItsExtendedWith>.swift` so in this case `InProcessTransport+Client.swift`. 

This makes things much easier to find: if you're looking for the in-process client transport then the obvious thing to search for is "InProcessTransport" because that's much more specific than "Client".